docs/design/windows-design.md

```diff
@@ -210,8 +210,17 @@ Kube-proxy userspace mode is configured to provide NodePort Service function. A
 "HNS Internal NIC" is provided to kube-proxy to configure Service addresses. The OpenFlow entries for the
 NodePort Service traffic on Windows are the same as those on Linux.
 
-The ClusterIP Service function is implemented by Antrea leveraging OVS. Antrea installs OpenFlow entries
-to select the backend endpoint and performs DNAT on the traffic.
+The ClusterIP Service function is implemented by AntreaProxy. Antrea installs routes from ClusterIP Services
+to a virtual IP (169.254.169.252) and a route from the virtual IP to antrea-gw0. There's an ARP responder flow
+for the virtual IP. The benefit of such design rather than routing a ClusterIP Service directly to antrea-gw0
+is that only one ARP responder flow is needed.
+
+For Services (hostNetwork: false), a packet to them will have its destination IP DNATed to endpoint Pod IP.
+For Services (hostNetwork: true), a packet to them will have its destination IP DNATed to peer Node IP. Then
+its source IP is SNATed to a virtual IP (169.254.169.253). Such SNAT is needed because for reply packet, if
+the destination IP is antrea-gw0 IP, then the packet will not enter OVS pipeline after routed to antrea-gw0.
+On Windows host, the request packet's source IP will be SNATed again to Node IP. The reply packets are the
```

Comment:
Did not get this sentence, when source IP will be SNAT'd to Node IP? In general, I feel you need to make sure your words are understandable, rather than trying to use less words.

Reply:
Added more description and a text graph.